ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/jackson/mixin/PartyMixIn.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.hmcts.cmc.ccd.jackson.mixin;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonUnwrapped;
+import uk.gov.hmcts.cmc.domain.models.Address;
+import uk.gov.hmcts.cmc.domain.models.legalrep.Representative;
+
+import java.time.LocalDate;
+
+public abstract class PartyMixIn {
```

Comment:
We are adding a lot of customisation just to prefix every attributes with `party`. May be worth investigation if we can so more Jackon magic here?